bounce/db/club.py

```diff
@@ -79,6 +98,7 @@ def search(session, query=None, page=0, size=MAX_SIZE):
 def insert(session, name, description, website_url, facebook_url,
            instagram_url, twitter_url):
     """Insert a new club into the Clubs table."""
+    """Any user should have the permission to insert"""
```

Comment:
Minor: this doc should look like this
```suggestion
    """Insert a new club into the Clubs table.
    Any user should have the permission to insert"""
```

You've got an extra pair of quotes in there.

Reply:
Sounds good, I'll make that change.